experiments/calibration/minimal_working_example/minimal_working_example.jl

```diff
@@ -0,0 +1,96 @@
+# This script describes the simplest possible calibration of ClimaLand,
+# with step by step explanations.
+# If one line or comment confuses you, or you believe it could be simplified,
+# reach out to Alexis Renchon.
+
+# We calibrate a single site simulation, using EnsembleKalmanProcesses.jl.
+# We calibrate the parameters g1 and g0, to better match canopy transpiration target data.
+
+# To perform this calibration we will need:
+# 1. a function returning our model transpiration output given the parameters we want to calibrate.
+# 2. the "truth" target data, to calibrate on.
+# 3. the prior distribution of these parameters.
+
+# Let's go!
+
+# Packages
+import ClimaLandSimulations.Fluxnet as CLS # to run the model
+import EnsembleKalmanProcesses as EKP # to perform the calibration
+import Random # we need it for ...
+
+# a function returning our model transpiration output given the parameters to calibrate
+function ClimaLand_transpiration(params) # params is a 2 element Array
+    g1, g0 = params
+
+    sv = CLS.run_fluxnet(
+                         "US-MOz";
+                         params = CLS.ozark_default_params(; conductance = CLS.conductance_ozark(; g1 = g1, g0 = g0)),
+                        )[1];
+
+    inputs = CLS.make_inputs_df("US-MOz")[1];
+
+    simulation_output = CLS.make_output_df("US-MOz", sv, inputs);
+
+    transpiration = simulation_output.transpiration .* 1e9 # 1e9 because very small otherwise
+
+    return transpiration
+end
+
+# "truth" target data to calibrate on
+transpiration_target = ClimaLand_transpiration([141.0, 0.0001]) # our default for Ozark, g1 in sqrt(Pa). This is equal to 4.46 sqrt(kPa).
+
+# parameters prior
+prior_g1 = EKP.constrained_gaussian("g1", 221, 100, 0, Inf) # mean of 221 sqrt(Pa) = 7 sqrt(kPa), std of 100 (3 kPa)
+# Ollie: why does it return μ=5.3, σ=0.4? Are the values transformed? Is this to be of same order of magnitude?
+prior_g0 = EKP.constrained_gaussian("g0", 0.00015, 0.0001, 0, Inf)
+prior = EKP.combine_distributions([prior_g1, prior_g0])
+
+# generate the initial ensemble and set up the ensemble Kalman inversion
+N_ensemble = 5
+N_iterations = 5
+rng_seed = 41 # Ollie: why 41?
```

Comment:
can erase? I dont think this matters too much

Reply:
In general - maybe should remove "Ollie" questions for now?

Reply:
Sorry I should've left a comment - this PR is not ready to be merged as I want to discuss these fews questions with Ollie first, but I think he is away at the moment. 

But I still thought it would be valuable to review at this stage